dtests/src/resources/scripts/putInto/create_and_load_testLTableWith30Columns.sql

```diff
@@ -0,0 +1,184 @@
+-- DROP TABLE IF ALREADY EXISTS --
+DROP TABLE IF EXISTS testL1;
+
+create table testL1 (id string NOT NULL, data string, data2 decimal, APPLICATION_ID string NOT NULL, ORDERGROUPID string,
+ PAYMENTADDRESS1 string, PAYMENTADDRESS2 string, PAYMENTCOUNTRY string, PAYMENTSTATUS string, PAYMENTRESULT string,
+ PAYMENTZIP string, PAYMENTSETUP string, PROVIDER_RESPONSE_DETAILS string, PAYMENTAMOUNT string, PAYMENTCHANNEL string,
+  PAYMENTCITY string, PAYMENTSTATECODE string,PAYMENTSETDOWN string,  PAYMENTREFNUMBER string, PAYMENTST string,
+ PAYMENTAUTHCODE string,  PAYMENTID string, PAYMENTMERCHID string, PAYMENTHOSTRESPONSECODE string,PAYMENTNAME string,
+ PAYMENTOUTLETID string, PAYMENTTRANSTYPE string,  PAYMENTDATE string, CLIENT_ID string, CUSTOMERID string)
+using column options (key_columns 'id,APPLICATION_ID', BUCKETS '128', REDUNDANCY '1');
```

Comment:
Have we added column delta in table definitions? If not the putInto sizes need to be higher e.g 10m and above

Reply:
No, COLUMN_MAX_DELTA_ROWS not added in table definitions.  We tried to create a table with the same properties user specified. But yes, we do have put into sizes higher than 10 million. The range is between 10 to 200 million for putInto. I can change the DDL to include COLUMN_MAX_DELTA_ROWS set to lower value.